air-quality-backend/scripts/run_forecast_etl.py

```diff
@@ -17,8 +18,14 @@ def main():
     cities = get_locations_by_type(AirQualityLocationType.CITY)
     logging.info(f"Finding data for {cities.__len__()} cities")
 
+    base_date = datetime.utcnow()
+    base_date_env = os.environ.get("FORECAST_INITIAL_DATE_TIME")
```

Comment:
Any reason we don't call this `FORECAST_BASE_TIME`? Just to keep the terminology consistent everywhere?